contrib/init/sysvinit-redhat/docker

```diff
@@ -56,7 +56,7 @@ start() {
         prestart
         printf "Starting $prog:\t"
         echo "\n$(date)\n" >> $logfile
-        "$unshare" -m -- $exec daemon $other_args &>> $logfile &
+        "$unshare" -m -- $exec daemon $other_args & >> $logfile &
```

Comment:
> Why not write ">> $logfile 2>&1 &" in this script?

Could've been an oversight. Perhaps you can update this PR with that, then @tianon and @phemmer can review it again